<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>pyPDFTKbuilder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_11">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_join">
       <attribute name="title">
        <string>&amp;Join Files</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Source PDF Document(s)</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="DropPDFQListWidget" name="join_listWidget"/>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string/>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QPushButton" name="add_pushButton">
                    <property name="toolTip">
                     <string>Add PDF(s) to the list</string>
                    </property>
                    <property name="text">
                     <string>A&amp;dd</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="remove_pushButton">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="toolTip">
                     <string>Remove item from list</string>
                    </property>
                    <property name="text">
                     <string>Remo&amp;ve</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="copy_pushButton">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="toolTip">
                     <string>Duplicate current item.</string>
                    </property>
                    <property name="text">
                     <string>CopyItem</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="mTop_pushButton">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="toolTip">
                     <string>Move item to top of list</string>
                    </property>
                    <property name="text">
                     <string>Top</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="mUp_pushButton">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="toolTip">
                     <string>Move item up in list.</string>
                    </property>
                    <property name="text">
                     <string>&amp;Up</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="mDown_pushButton">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="toolTip">
                     <string>Move item down in list</string>
                    </property>
                    <property name="text">
                     <string>&amp;Down</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="mBottom_pushButton">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="toolTip">
                     <string>Move item to bottom of list</string>
                    </property>
                    <property name="text">
                     <string>Bottom</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="clear_pushButton">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="toolTip">
                     <string>Clear ALL items from list</string>
                    </property>
                    <property name="text">
                     <string>Clear</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="sort_pushButton">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="toolTip">
                     <string>Sort the list</string>
                    </property>
                    <property name="text">
                     <string>Sor&amp;t</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="shuffleCat_pushButton">
                    <property name="text">
                     <string>Catenate</string>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="saveAs_pushButton">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="toolTip">
                     <string>Save As ...</string>
                    </property>
                    <property name="text">
                     <string>Save &amp;As ...</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_split">
       <attribute name="title">
        <string>&amp;Split Files</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>44</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="20,15,20">
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="burstPdfSourcePushButton">
            <property name="toolTip">
             <string>Select PDF to split</string>
            </property>
            <property name="text">
             <string>Source PDF File...</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="burstPdfLabel">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>foo</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>44</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>323</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="burstPdfSavePushButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Save</string>
             </property>
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>322</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_bsn">
       <attribute name="title">
        <string>&amp;Background/Stamp/Number</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QWidget" name="protectionWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QGroupBox" name="groupBox_5">
              <property name="title">
               <string>User Password</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <property name="topMargin">
                <number>1</number>
               </property>
               <property name="bottomMargin">
                <number>1</number>
               </property>
               <item>
                <widget class="QLineEdit" name="passUserLineEdit">
                 <property name="clearButtonEnabled">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_7">
              <property name="title">
               <string>Owner Password</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="topMargin">
                <number>1</number>
               </property>
               <property name="bottomMargin">
                <number>1</number>
               </property>
               <item>
                <widget class="QLineEdit" name="passOwnerLineEdit">
                 <property name="clearButtonEnabled">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_8">
            <property name="title">
             <string>Encryption</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QRadioButton" name="noneRadioButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>None</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rc4_40RadioButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>RC4 40</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rc4_128RadioButton_">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>RC4 128</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_9">
            <property name="title">
             <string>Allow</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_13">
               <item>
                <widget class="QCheckBox" name="allowPrintCheckBox">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Printing</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="allowCopyContentsCheckBox">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Copy Contents</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="allowModifyContentsCheckBox">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Modify Contents</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="allowModifyAnnotationsCheckBox">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Modify Annotations</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_14">
               <item>
                <widget class="QCheckBox" name="allowDegradedPrintingCheckBox">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Degraded Printing</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="allowScreenreadersCheckBox">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Screenreaders</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="allowAssemblyCheckBox">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Assembly</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="allowFillInCheckBox">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Fill in</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0">
            <item>
             <widget class="QPushButton" name="permissionsPushButton">
              <property name="text">
               <string>Permissions</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionJoin_Files"/>
    <addaction name="actionSplit_Files"/>
    <addaction name="actionBackground_Stamp_Number"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Document_Protection"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionJoin_Files">
   <property name="text">
    <string>Join Files</string>
   </property>
  </action>
  <action name="actionSplit_Files">
   <property name="text">
    <string>Split Files</string>
   </property>
  </action>
  <action name="actionBackground_Stamp_Number">
   <property name="text">
    <string>Background/Stamp/Number</string>
   </property>
  </action>
  <action name="actionShow_Document_Protection">
   <property name="text">
    <string>Show Document Protection</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DropPDFQListWidget</class>
   <extends>QListWidget</extends>
   <header>DropPDFQListWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>permissionsPushButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>protectionWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>536</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
